src/Sylius/Bundle/CoreBundle/spec/ImportExport/Processor/UserProcessorSpec.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\CoreBundle\ImportExport\Processor;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\ImportExport\Converter\DateConverter;
+use Prophecy\Argument;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class UserProcessorSpec extends ObjectBehavior
+{
+    function let(DateConverter $dateConverter)
+    {
+        $this->beConstructedWith($dateConverter);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\CoreBundle\ImportExport\Processor\UserProcessor');
+    }
+
+    function it_implements_user_processor_interface()
+    {
+        $this->shouldImplement('Sylius\Bundle\CoreBundle\ImportExport\Processor\UserProcessorInterface');
+    }
+
+    function it_process_array_of_users_to_writers_friendly_form(
+        $dateConverter,
+        \DateTime $birthdayDate,
+        \DateTime $createdAtDate,
+        \DateTime $credentialsExpireAtDate,
+        \DateTime $customerCreatedAtDate,
+        \DateTime $customerDeletedAtDate,
+        \DateTime $customerUpdatedAtDate,
+        \DateTime $deletedAtDate,
+        \DateTime $expiresAtDate,
+        \DateTime $lastLoginDate,
+        \DateTime $passwordRequestedAtDate,
+        \DateTime $updatedAtDate
```

Comment:
As your mocks don't have any behaviour and are all the same, you can use only one instead of 11 :)